### PR TITLE
Add YAML lint check

### PR DESCRIPTION
**Pull request by gagiuntoli**

### CODE REVIEW

File: prod/onChainData128.8-validator1.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ETH-USD:
   discrepancy: 0.5 # allowed discrepancy level for price signing [%]
-  precision: 8 # we can choose any precision for value, but it will be stored with 8 decimals always
```
Comment (DZariusz) [inline prod/onChainData128.8-validator1.yaml:3]: we can not have long comments?

Comment (gagiuntoli) [reply prod/onChainData128.8-validator1.yaml:3]: it is set to 80 characters by default, would you like that increase that limit?

Comment (DZariusz) [reply prod/onChainData128.8-validator1.yaml:3]: well, let's use default

File: prod/onChainData128.8-validator1.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
+---
```
Comment (DZariusz) [inline prod/onChainData128.8-validator1.yaml:1]: please copy this file to separate PR so wecan merge it before we merge this PR

**Review by DZariusz — APPROVED** [review]
please copy

**Pull request merged**